x-pack/plugins/alerting/common/routes/rule/response/types/v1.ts

```diff
@@ -13,6 +13,7 @@ import {
   ruleLastRunSchemaV1,
   monitoringSchemaV1,
 } from '..';
+import type { RuleTypeParams, SanitizedRule } from '../../../../../server/types';
```

Comment:
I tried it, but it does not work. In create rule you call `transformRuleToRuleResponse`, so we get something different from sanitizedRule. One difference I can see it's bulkDelete returns camel case, but create rule returns  snake case.

Reply:
@JiaweiWu We with @XavierM decided that because _bulk_delete is internal we can change it. I've rewritten it the same way _bulk_edit is. Now it's returning the same rules type.